roleGroup.py

```diff
@@ -81,13 +81,13 @@ def fromJson(self, data: dict, emojis: Tuple[Emoji], manual_load=False):
                 # Only create new roles if we're not loading data manually from
                 # the command channel
                 role = Role(roleData["name"], roleEmoji,
-                            roleData["displayName"])
+                            self.getCorrectedName(roleData)) # change method to roleData["show_name"]
```

Comment:
What is happening here? Why do you want to load getCorrectedName here?

Reply:
That's a leftover from #47 where some of the database values got changed. That change should disappear from this PR after #47 gets merged

Reply:
Cool glad it will get removed after the big change.

Reply:
big change

wait until you see my newest pull request. thats some chaos

Reply:
uh oh

Reply:
Removed extra commits from the history